radlab-ui/webapp/src/pages/api/regions.ts

```diff
@@ -0,0 +1,55 @@
+import { NextApiRequest, NextApiResponse } from "next"
+
+import axios from "axios"
+import { generateAccessToken } from "@/utils/api"
+import { envOrFail } from "@/utils/env"
+
+const projectId = envOrFail(
+  "NEXT_PUBLIC_GCP_PROJECT_ID",
+  process.env.NEXT_PUBLIC_GCP_PROJECT_ID,
+)
+
+const getRegions = async (_: NextApiRequest, res: NextApiResponse) => {
+  const token = await generateAccessToken()
+  const data = await axios({
+    method: "GET",
+    url: `https://compute.googleapis.com/compute/v1/projects/${projectId}/regions`,
+    headers: {
+      Authorization: `Bearer ${token}`,
+    },
+  })
+  const itemsPage = data.data.items
+  if (!itemsPage) {
+    return res.status(200).json({ regions: itemsPage })
```

Comment:
I don't think a `200` is appropriate when something fails in the previous call.

Reply:
And let's return `regions: null` instead since it's more clear than an implicit `undefined`